lib/rules/space-infix-ops.js

```diff
@@ -75,26 +116,29 @@ module.exports = {
          * @private
          */
         function report(mainNode, culpritToken) {
+            const operator = culpritToken.value;
+            const must = spacesYes[operator] ? "must" : "must not";
+
             context.report({
                 node: mainNode,
                 loc: culpritToken.loc.start,
-                message: "Infix operators must be spaced.",
+                message: `${operator} operator ${must} be surrounded by spaces.`,
```

Comment:
I know this will seem unintuitive in an ES6 codebase that uses template strings everywhere else, but could you please use a non-template string and `{{operator}}` and `{{must}}`, and then add a `data` object `data: { operator, must }`?

The reason for this is because I'm hoping we can eventually centralize all messages into rule metadata, and we would leave them as strings with placeholders. This is good for eventual internationalization, as well as using rule metadata to generate rules' possible messages for website searching. When we use a template string, the placeholders are substituted before we can use the string, so those other purposes would be prevented.

This only applies to `context.report()` violation messages-- please feel free to use template strings everywhere else.

See [here](http://eslint.org/docs/developer-guide/working-with-rules#contextreport) for more info.